render/gles2/shaders.c

```diff
@@ -3,100 +3,73 @@
 
 // Colored quads
 const GLchar quad_vertex_src[] =
-"uniform mat4 proj;"
-"uniform vec4 color;"
-"attribute vec2 pos;"
-"attribute vec2 texcoord;"
-"varying vec4 v_color;"
-"varying vec2 v_texcoord;"
-"mat4 transpose(in mat4 inMatrix) {"
-"    vec4 i0 = inMatrix[0];"
-"    vec4 i1 = inMatrix[1];"
-"    vec4 i2 = inMatrix[2];"
-"    vec4 i3 = inMatrix[3];"
-"    mat4 outMatrix = mat4("
-"                 vec4(i0.x, i1.x, i2.x, i3.x),"
-"                 vec4(i0.y, i1.y, i2.y, i3.y),"
-"                 vec4(i0.z, i1.z, i2.z, i3.z),"
-"                 vec4(i0.w, i1.w, i2.w, i3.w)"
-"                 );"
-"    return outMatrix;"
```

Comment:
Where did this logic go?

Reply:
https://github.com/swaywm/wlroots/pull/290/commits/7170170a5778a0bbb37827f85882cde4c7762ec5#diff-650ac2c117ab1d2399640eab8ba3ab74R170

Reply:
Hah, derp. Good catch.